src/NDItk/multigroup/CrossSection/test/CrossSection.test.cpp

```diff
@@ -0,0 +1,130 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_floating_point.hpp>
+using Catch::Matchers::WithinRel;
+
+// what we are testing
+#include "NDItk/multigroup/CrossSection.hpp"
+
+// other includes
+
+// convenience typedefs
+using namespace njoy::NDItk;
+using CrossSection = multigroup::CrossSection;
+
+std::string chunk();
+std::vector< double > data();
+void verifyChunk( const CrossSection& );
+std::vector< double > dataWithInsufficientLength();
+
+SCENARIO( "CrossSection" ) {
+
+  GIVEN( "valid data for a CrossSection instance" ) {
+
+    std::string record = chunk();
+
+    WHEN( "the data is given explicitly" ) {
+
+      int reaction = 2;
+      double qvalue = 0.0;
+      std::vector< double > values = { 10., 20., 30., 40., 50., 60., 70. };
```

Comment:
Since we override the print function, yes, this is a good idea.

Changes made to fix/review. I also added additional digits there for the RealListRecord test.